cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+    dataloader = DataLoader(vuln, batch_size=1)
+
+    return dataloader
+
+def decode_output(outputs):
+
+    output_decoded = []
+
+    for index, i in enumerate(outputs.values()):
+
+        with open(f'utils/encoder_{OUTPUTS_NAME[index]}.pickle', 'rb') as handle:
+            encoder = pickle.load(handle)
```

Ruff

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)